idemp-base/src/org/bandahealth/idempiere/base/model/MBandaSetup.java

```diff
@@ -1359,6 +1361,54 @@ public boolean configureClientUsers() {
 		});
 		return true;
 	}
+	
+	/**
+	 * Create default business partners for new clients
+	 * @return
+	 */
+	public boolean createDefaultBusinessPartners() {
+		// get business partner group
+		MBPGroup businessPartnerGroup = new Query(this.context, MBPGroup.Table_Name, MBPGroup.COLUMNNAME_AD_Client_ID + "=?", getTransactionName())
```

Comment:
I'm afraid this will get the `Standard` business partner group instead of the one that's supposed to be assigned. I'd recommend getting the group that's assigned to the business partner (in the config client). You can see how this is done in the `addDefaultChargeTypes` method - it returns a map of the old ID (that was assigned to the config client) to the new charge (that was saved on the created client).